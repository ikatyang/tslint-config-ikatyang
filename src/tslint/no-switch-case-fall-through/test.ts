// good

switch (1) {
  case 1:
    console.log(1);
  // falls through
  default:
    console.log(2);
}

// bad

switch (1) {
  case 1:
    console.log(1);
  default:
    console.log(2);
}
