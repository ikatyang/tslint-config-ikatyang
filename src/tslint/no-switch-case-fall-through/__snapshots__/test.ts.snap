// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lint 1`] = `
// good

switch (1) {
  case 1:
    console.log(1);
  // falls through
  default:
    console.log(2);
}

// bad

switch (1) {
  case 1:
    console.log(1);
  default:
  ~~~~~~~  [expected a 'break' before 'default']
    console.log(2);
}

`;
